torchvision/prototype/transforms/functional/_meta.py

```diff
@@ -183,12 +183,8 @@ def clamp_bounding_box(
     return convert_format_bounding_box(xyxy_boxes, BoundingBoxFormat.XYXY, format)
 
 
-def _split_alpha(image: torch.Tensor) -> Tuple[torch.Tensor, torch.Tensor]:
-    return image[..., :-1, :, :], image[..., -1:, :, :]
-
-
 def _strip_alpha(image: torch.Tensor) -> torch.Tensor:
-    image, alpha = _split_alpha(image)
+    image, alpha = torch.tensor_split(image, indices=(-1,), dim=-3)
```

Comment:
IMO we don't need a private one liner method for something Torch has a built in method.

Reply:
Non-blocking: I would prefer it since it is a little more expressive. Plus, since we have a `_add_alpha` function it would be consistent to have the complementary `_split_alpha` as well. With this change we have three identical, non-trivial calls of `torch.tensor_split`. At a glance, it is harder to see that these three actually do exactly the same.

Reply:
OK I'll restore `_split_alpha.

Reply:
Restoring this causes more [harm](https://github.com/pytorch/vision/actions/runs/3320465913/jobs/5486881222) than good. JIT is complaining that it's not longer a tuple, which means I would have to split into 2 lines and return as tuple to please it. I don't think it's worth this.